<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextureForm</class>
 <widget class="QWidget" name="TextureForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>246</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Texture Form</string>
  </property>
  <layout class="QFormLayout" name="form_layout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <property name="horizontalSpacing">
    <number>4</number>
   </property>
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label_name">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="name"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_image_format">
     <property name="text">
      <string>Image Format:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="image_format">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_image_size">
     <property name="text">
      <string>Image Size:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="image_size">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_image_levels">
     <property name="text">
      <string>Image Levels:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="image_levels">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_palette_format">
     <property name="text">
      <string>Palette Format:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="palette_format">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_palette_size">
     <property name="text">
      <string>Palette Size:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="palette_size">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_wrap_s">
     <property name="text">
      <string>Wrap S:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QComboBox" name="wrap_s"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_wrap_t">
     <property name="text">
      <string>Wrap T:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QComboBox" name="wrap_t"/>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_minification_filter">
     <property name="text">
      <string>Min. Filter:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QComboBox" name="minification_filter"/>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_magnification_filter">
     <property name="text">
      <string>Mag. Filter:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QComboBox" name="magnification_filter"/>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_minimum_lod">
     <property name="text">
      <string>Min. LOD:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QDoubleSpinBox" name="minimum_lod">
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_maximum_lod">
     <property name="text">
      <string>Max. LOD:</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QDoubleSpinBox" name="maximum_lod">
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_lod_bias">
     <property name="text">
      <string>LOD Bias:</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QDoubleSpinBox" name="lod_bias">
     <property name="minimum">
      <double>-4.000000000000000</double>
     </property>
     <property name="maximum">
      <double>3.990000000000000</double>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="label_unknown0">
     <property name="text">
      <string>Unknown 0:</string>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="QSpinBox" name="unknown0">
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="label_unknown1">
     <property name="text">
      <string>Unknown 1:</string>
     </property>
    </widget>
   </item>
   <item row="14" column="1">
    <widget class="QSpinBox" name="unknown1">
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
   </item>
   <item row="15" column="0">
    <widget class="QLabel" name="label_unknown2">
     <property name="text">
      <string>Unknown 2:</string>
     </property>
    </widget>
   </item>
   <item row="15" column="1">
    <widget class="QSpinBox" name="unknown2">
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
